lisa/features/disk_ephemeral.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from lisa.feature import Feature
+
+FEATURE_NAME_DISK = "DiskEphemeral"
+
+
+class DiskEphemeral(Feature):
+    @classmethod
+    def name(cls) -> str:
+        return FEATURE_NAME_DISK
+
+    @classmethod
+    def can_disable(cls) -> bool:
+        return True
+
+    def _switch(self, enable: bool) -> None:
+        raise NotImplementedError()
+
+    def disable(self) -> None:
+        self._switch(False)
+
+    def enable(self) -> None:
+        self._switch(True)
+
+    def enabled(self) -> bool:
+        return True
+
+    @staticmethod
+    def get_disk_id() -> str:
```

Comment:
What's reason is this a static method?

Reply:
This method doesn't depend on the state of the class and neither it seems to fit `classmethod`. I think the other methods in feature class modify the enabled/disabled state which is why they aren't static.